stablehlo/integrations/c/StablehloDialect.h

```diff
@@ -22,6 +22,9 @@ extern "C" {
 
 MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(Stablehlo, stablehlo);
 
+// Get the current version of StableHLO API's. Incremented with all changes.
+MLIR_CAPI_EXPORTED int getStablehloApiVersion();
```

Comment:
Do we need to expose this as part of our C API? Fwiw, we don't do this for `getCurrentVersion`.

Reply:
This would remove the need for users of StablehloModule.cpp from taking another dependency on `:version`. We got around this for those other API's by placing them in `Serialization.h`, which may not be the proper location for them.

We could move all "get*version" functions to `Version.h` and require the additional dependency from users of StablehloModule. Perhaps that's a more proper design.